.gitignore

```diff
@@ -1,2 +1,48 @@
 node_modules
 .DS_Store
+### JetBrains template
```

Comment:
is there a reason we have to put this in here? your IDE specific things should go into your own global `.gitignore` file